Traceback (most recent call last):
  File "/local/home/ceriotti/lavoro/code/i-pi/drivers/py/pes/__init__.py", line 16, in <module>
    module = importlib.import_module("." + module_name, __package__)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/local/home/ceriotti/lavoro/code/i-pi/drivers/py/pes/metatensor.py", line 15, in <module>
    import metatensor.torch
  File "/home/ceriotti/local/lib/python3.10/site-packages/metatensor/torch/__init__.py", line 5, in <module>
    from ._c_lib import _load_library
  File "/home/ceriotti/local/lib/python3.10/site-packages/metatensor/torch/_c_lib.py", line 14, in <module>
    if not version_compatible(metatensor.__version__, BUILD_METATENSOR_CORE_VERSION):
AttributeError: module 'metatensor' has no attribute '__version__'
/local/home/ceriotti/lavoro/code/i-pi/bin/i-pi-py_driver:159: DeprecationWarning: Conversion of an array with ndim > 0 to a scalar is deprecated, and will error in future. Ensure you extract a single element from your array before performing this operation. (Deprecated NumPy 1.25.)
  send_data(sock, np.float64(pot))
!! Could not import PES module metatensor !!
Received exit message from i-PI. Bye bye!
